x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/validation.ts

```diff
@@ -203,6 +257,12 @@ function getMessageFromId<K extends ErrorTypes>({
         values: { operation: out.operation, count: out.count, params: out.params },
       });
       break;
+    case 'tooManyQueries':
+      message = i18n.translate('xpack.lens.indexPattern.formulaOperationDoubleQueryError', {
+        defaultMessage: 'The operation {operation} contains too many queries',
+        values: { operation: out.operation },
```

Comment:
Can we make this error more specific, since we already have a case for duplicate parameters? I suggest "Use only one of kql= or lucene=, not both"